File: cpp/atcoder/arc113a.cpp
#include<bits/stdc++.h>
#define int long long
using namespace std;
int ans=0;
int n;
signed main(){
        cin>>n;
        for(int i=1;i<=n;i++){
                for(int j=1;i*j<=n;j++){
                        ans+=max(1LL,n/(i*j));
                }
        }
        cout<<ans<<'\n';
}

File: cpp/atcoder/arc113a.py
n=int(input())
ans=0
for i in range(1,n+1):
    for j in range(1,n+1):
        if i*j>k:
            break
        for k in  range(1,n+1):
            if i*j*k>n:
                break
            ans+=1
print(ans)
